src/client/datascience/kernel-launcher/kernelProcess.ts

```diff
@@ -343,7 +343,15 @@ export class KernelProcess implements IKernelProcess {
                 this.processExecutionFactory.create(this.resource),
                 this.kernelEnvVarsService.getEnvironmentVariables(this.resource, this.launchKernelSpec)
             ]);
-            exeObs = executionService.execObservable(executable, this.launchKernelSpec.argv.slice(1), {
+
+            // Add quotations to arguments if they have a blank space in them
```

Comment:
I think I might add that you didn't want to quote everything because it can break a kernel to have all arguments quoted.